Update the FAQ for evil-leader rebinding

StreakyCobra opened the pull request:
As `SPC l` and `SPC L` are now inverted, the FAQ has to be updated. The
question is also moved to "How do I …" where it belongs.

TheBB commented:
Does not https://github.com/syl20bnr/spacemacs/pull/3673 do this? Well, it doesn't move it, but...

StreakyCobra commented:
I didn't saw it before, sorry. Indeed it corrects `l` to `L`, but in this one I also changed the code and added a few more information about it.

I don't really care if it gets merged or not, I helped someone on the chat and quickly did a PR from the code, so if you don't think it's necessary just close it, it's fine :smile:

robbyoconnor commented:
:+1:

TheBB commented:
This should be updated. The example isn't really that relevant any more, I think.

StreakyCobra commented:
Updated? You mean merged, or you mean I should update my PR?

TheBB commented:
The PR I linked to was merged, which means this one now conflicts. And avy-line is on `SPC y` now.

TheBB commented:
Mostly concerned about the latter, of course. I could resolve the conflict myself.

StreakyCobra commented:
Oh no it's fine, I didn't saw the `SPC y` merged. I'll update it :-)

StreakyCobra commented:
I'll ping you when done

TheBB commented:
You didn't ping me but I merged it anyway. Thanks! Cherry-picked in develop. You can safely delete your branch.
